packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,67 @@
+# Amount <!-- omit in toc -->
+
+Amount component is a UI element that displays and formats various currency values in a specified format.
+
+<img  src="./amount-thumbnail.png"  width="380" />
+
+## Design
+
+- [Figma - Amount](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=3peAz8A2n2Gw4WMl-1)
+
+## API
+
+| Prop           | Type                                                               | Default     | Description                                                                    | Required |
+| -------------- | ------------------------------------------------------------------ | ----------- | ------------------------------------------------------------------------------ | -------- |
+| children       | `string`                                                           | `undefined` | The value to be rendered within the component                                  | ✅       |
+| intent         | `default`, `positive`, `negative`, `notice`, `information`, `blue` | `default`   | The variant of the amount to be rendered                                       |          |
+| contrast       | `low`, `high`                                                      | `low`       | The contrast of the amount to be rendered                                      |          |
+| fontWeight     | `bold`, `regular`                                                  | `bold`      | The weight of the amount to be rendered                                        |          |
+| hasDecimals    | `true`, `false`                                                    | `true`      | Indicates whether the amount has or can have decimals                          |          |
+| hasTextPostfix | `true`, `false`                                                    | `true`      | Indicates whether a text postfix should be used                                |          |
+| size           | `3xl`, `2xl`, `xl`, `large`, `medium`, `small`                     | `3xl`       | The size of the amount to be rendered                                          |          |
+| isStyled       | `true`, `false`                                                    | `true`      | Highlight the main amount by making the prefix symbol and decimal digits small |          |
+| screenSize     | `desktop`, `mobile`                                                | `desktop`   | Provides the size of the component based on the platform                       |          |
+
+### Sample Usage
+
+```jsx
+import { Amount } from '@razorpay/components';
+
+<Amount
+  size="2XL"
+  isStyled="true"
+  fontWeight="regular"
+  hasDecimals="true"
+  hasTextPostfix="true"
+  intent="information"
+  contrast="high"
+>
+  1000.27
+</Amount>;
+```
+
+## Open Questions
+
+#### Dev
+
+- Should we call it `variant` or `intent`, `isStyled` or something more apt?
+- Should we separate the amount with commas based on the Indian currency format?
```

Comment:
Are we only supporting 1 currency format (INR)? Or should we allow the consumer to decide what format it should be?

Reply:
For now, we will be supporting only INR. Going forward we will add more currencies. One of the major problems right now is we use ₹ and INR interchangeably. But in Malaysia we only have RM and there is no symbol associated with it. Hence we are staying away from this. Will solve this once the use-case arrives.

Reply:
Just leaving my thoughts here. 
Dashboard is currently preparing for launching in Malaysia. I am not sure about the timelines but I do see a big chunk of code already being merged to prepare for it, so the use-case might arrive sooner. Perhaps we can also support Malaysia along with INR for now.

for reference this is one of the PR's which was merged couple of months back
https://github.com/razorpay/dashboard/pull/11454/files

Reply:
Agreed. We should start thinking about currency support from now. We can limit it to just INR for now and add more later